lldb/test/API/functionalities/archives/TestBSDArchives.py

```diff
@@ -25,7 +25,6 @@ def setUp(self):
         oslist=["windows"],
         bugnumber="llvm.org/pr24527.  Makefile.rules doesn't know how to build static libs on Windows",
     )
-    @expectedFailureAll(remote=True)
```

Comment:
And this works now because...? Because it in a remote context it finds the tool in the llvm build dir where before it wouldn't?

Reply:
Just need a basic explanation of this then I'll approve the PR.

Reply:
I thought it failed because of the missing tool, but it unexpectedly passed on my machine on mainline.

Reply:
https://lab.llvm.org/staging/#/builders/195/builds/2710/steps/16/logs/XPASS__lldb-api__TestBSDArchives_py

Reply:
Ok fair enough, maybe it's because the `llvm-whatever` is better than the ancient distro tool.

But if this is already UPASS-ing without this PR then it should be put in its own commit (if you can push direct) or PR (if not).

Reply:
Fixed